includes/wcs-renewal-functions.php

```diff
@@ -62,6 +62,18 @@ function wcs_order_contains_renewal( $order ) {
 	return apply_filters( 'woocommerce_subscriptions_is_renewal_order', $is_renewal, $order );
 }
 
+/**
+ * Determines if a given order is the subscription's latest renewal order.
+ *
+ * @param $order WC_Order The order object.
+ * @param $subscription WC_Subscription The subscription object.
+ * @return bool Whether the order is the latest renewal order of the provided subscription.
+ */
+function wcs_order_is_last_renewal_of_subscription( $order, $subscription ) {
+	$last_renewal_order = wcs_get_renewal_orders_sorted_by( $subscription, 'date_created' );
```

Comment:
```suggestion
	$last_renewal_order = wcs_get_last_renewal_order( $subscription );
```

I think you changed this function by mistake in your last changes. 

It's throwing this error:

```
PHP Fatal error:  Uncaught Error: Call to a member function get_id() on array in wp-content/plugins/woocommerce-subscriptions-core/includes/wcs-renewal-functions.php:74
```